js/apps/account-ui/src/api/representations.ts

```diff
@@ -87,16 +88,9 @@ export interface UserProfileMetadata {
   attributes: UserProfileAttributeMetadata[];
 }
 
-export interface UserRepresentation {
-  id: string;
-  username: string;
-  firstName: string;
-  lastName: string;
-  email: string;
-  emailVerified: boolean;
+export type UserRepresentation = UserFormFields & {
```

Comment:
`UserFormFields` is a different type than `UserRepresentation`, if we need a `UserRepresentation` we should use the one exported by the Admin Client directly.

Reply:
atm account-ui has it's own "client" and it doesn't need or use the admin rest endpoints, seems wrong to make it dependent on that just to share a domain model representation

Reply:
I think it's fine to import the types from the Admin Client, they will get compiled away during bundling anyways.